content/en/users/compute/cloud-compute/dynamic-dns/_index.md

```diff
@@ -19,6 +19,9 @@ modifying server/client configurations (federated approach), can request valid
 server certificates in advance (critical for security) and many other
 advantages.
 
+A short demonstration video is available at
+[fedcloud.eu YouTube channel](https://www.youtube.com/watch?v=dk4VYT2VFmU).
```

Comment:
Hi @tdviet, all the work you do is great, but I'm a bit surprised by all this fedcloud.eu branding, I fail to understand why it's not much linked to the EGI Federation activities and branding.
At some point it was agreed to close some old GitHub organisations (like EGI-FCTF) to try to have a more coherent/clear branding and ease repositories discovery, collaboration, enforce some common best practices and so on, by having an EGI-Federation organisation meant to collect work related to the EGI Federation, including for EGI Federated Cloud (or EGI Cloud Compute as its current official name).
And now there is dedicated fedcloud.eu branding, with a separate GitHub organisation, a dedicated youtube channel, and some separate domains (including references to a webpage only showing the default HTTPD page: https://fedcloud.eu). All this without any apparent coordination/discussion, it's a bit confusing. If there are issues/concerns leading to this situation, and as apparently we fail to understand this, maybe we should discuss this more openly at some point.

Reply:
Hi @gwarf,

for this specific PR, you can copy the video from fedcloud.eu channel to EGI channel and change the link in the docs accordingly if that is convenient for you.

For the fedcloud.eu branding, it needs a longer explanation and GitHub is not the suitable place for such a discussion. I will send a separate email latter for the issue.